src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts

```diff
@@ -32,6 +32,8 @@
 
         public depthOfFieldPostProcess: Nullable<PostProcess> = null;
 
+        public fxaaPostProcess: Nullable<FxaaPostProcess> = null;

```

Comment:
I think the doc check will fail on this one ;)

Reply:
That means everything must be commented in the pipeline :)